index.bs

```diff
@@ -179,12 +179,12 @@ General Requirements & Brands {#brands}
 -----------------------------
 
 A file conformant to this specification satisfies the following:
-- It SHALL conform to the normative requirements of [[!ISOBMFF]]
-- It SHALL have the <dfn value export for="ISOBMFF Brand">av01</dfn> brand among the compatible brands array of the FileTypeBox
-- It SHALL contain at least one track using an [=AV1SampleEntry=]
-- It SHOULD indicate a structural ISOBMFF brand among the compatible brands array of the FileTypeBox, such as 'iso6'
-- It MAY indicate CMAF brands as specified in [[#cmaf]]
-- It MAY indicate other brands not specified in this document provided that the associated requirements do not conflict with those given in this specification
+- <assert>It SHALL conform to the normative requirements of [[!ISOBMFF]]</assert>
+- <assert>It SHALL have the <dfn value export for="ISOBMFF Brand">av01</dfn> brand among the compatible brands array of the FileTypeBox</assert>
+- <assert>It SHALL contain at least one track using an [=AV1SampleEntry=]</assert>
+- <assert>It SHOULD indicate a structural ISOBMFF brand among the compatible brands array of the FileTypeBox, such as 'iso6'</assert>
+- <assert>It MAY indicate CMAF brands as specified in [[#cmaf]]</assert>
+- <assert>It MAY indicate other brands not specified in this document provided that the associated requirements do not conflict with those given in this specification</assert>
 
 Parsers SHALL support the structures required by the <code>'iso6'</code> brand and MAY support structures required by further ISOBMFF structural brands.
```

Comment:
Thanks. I did not wrap it with an assert because the requirement applies to Parser and I could not think of conformance files to exercise them, at the time. Now that I think of it, it seems we could have valid files with `iso6` and some with `isoX` with X>6. I'll update the PR.